Fixing FTBFS on power:
https://bugzilla.redhat.com/show_bug.cgi?id=2120974

Upstream issue: https://github.com/xianyi/OpenBLAS/issues/3738
Upstream fix: https://github.com/xianyi/OpenBLAS/pull/3718

commit d9dc015cfc78fc32f555995a89d6957ef0184ea2
Author: Martin Kroeker <martin@ruby.chemie.uni-freiburg.de>
Date:   Mon Aug 8 14:52:10 2022 +0200

    Use blasint for INTERFACE64 compatibility

diff --git a/test/compare_sgemm_sbgemm.c b/test/compare_sgemm_sbgemm.c
index a2c358cf..d4b59145 100644
--- a/test/compare_sgemm_sbgemm.c
+++ b/test/compare_sgemm_sbgemm.c
@@ -76,9 +76,9 @@ float16to32 (bfloat16_bits f16)
 int
 main (int argc, char *argv[])
 {
-  int m, n, k;
+  blasint m, n, k;
   int i, j, l;
-  int x;
+  blasint x;
   int ret = 0;
   int loop = 100;
   char transA = 'N', transB = 'N';

commit 3d338b57de1837f1e2264a1262a9ee9203f31c6c
Author: Martin Kroeker <martin@ruby.chemie.uni-freiburg.de>
Date:   Mon Aug 8 17:09:45 2022 +0200

    remove spurious loops

diff --git a/test/compare_sgemm_sbgemm.c b/test/compare_sgemm_sbgemm.c
index d4b59145..276fecae 100644
--- a/test/compare_sgemm_sbgemm.c
+++ b/test/compare_sgemm_sbgemm.c
@@ -112,7 +112,6 @@ main (int argc, char *argv[])
 	      &m, BB, &k, &beta, CC, &m);
       for (i = 0; i < n; i++)
 	for (j = 0; j < m; j++)
-	  for (l = 0; l < k; l++)
 	    if (fabs (CC[i * m + j] - C[i * m + j]) > 1.0)
 	      ret++;
       if (transA == 'N' && transB == 'N')
@@ -126,7 +125,6 @@ main (int argc, char *argv[])
 		}
 	  for (i = 0; i < n; i++)
 	    for (j = 0; j < m; j++)
-	      for (l = 0; l < k; l++)
 		if (CC[i * m + j] != DD[i * m + j])
 		  ret++;
 	}
